src/bp/core/config/botpress.config.ts

```diff
@@ -569,3 +575,23 @@ export interface EventCollectorConfig {
    */
   ignoredEventProperties: string[]
 }
+
+interface ActionServersConfig {
+  localActionServer: {
+    /**
+     * Port on which the local Action Server listens
+     * @default 4000
+     */
+    port: number
+    /**
+     * Whether or not the enable the local Action Server
+     * @default true
+     */
+    enabled: boolean
+  }
+  /**
+   * The list of remote Action Servers
+   * @default []
+   */
+  remoteActionServers: ActionServer[]
```

Comment:
Since you are already in the namespace "ActionServers", we could name them "localServer" and "remoteServers", or even better "local" and "remote" (with a description to say that it's a remote action server ..)

Reply:
Done in f087d7503a1e2d4a73a051c87795000d46b18020